spec/game_spec.rb

```diff
@@ -164,7 +164,7 @@
       GameFactory.new
       GameFactory.create_games("./fixtures/games_fixture.csv")
 
-      expect(Game.average_goals_per_game).to eq(3)
+      expect(Game.average_goals_per_game).to eq(3.75)
```

Comment:
I knew what you were describing in the PR but always nice to see the changes side by side here. Nice work.